metaflow/mflog/mflog.py

```diff
@@ -137,7 +137,7 @@ def line_iter(logblob):
             res = MFLogline(
                 False, None, MISSING_TIMESTAMP_STR, None, None, line, MISSING_TIMESTAMP
             )
-            yield res.utc_tstamp_str, res
+            yield res.utc_tstamp_str.encode("utf-8"), res
```

Comment:
just some thoughts on the use of the timestamps, am I reading it correctly that the utc_tstamp_str is only used internally for sorting the log lines? I had a slight concern if encoding the tstamp_str only for the `yield` would lead to issues, as we're also yielding the `res`, which contains the non-bytestring version of the timestamp. 

This doesn't seem to be used anywhere else in the codebase for now, but it is somewhat of a sharp edge. Would it make sense to encode the tstamp_str already for the Tuple?

Not a blocker for merging :)